.vib/concourse/goss/web/vars.yaml

```diff
@@ -0,0 +1,34 @@
+secrets:
+  localUser: "bitnamiUser"
```

Comment:
Is there a need to have some of these values between double quotes? Yaml should be treating them as strings in the `goss.yaml` file even without them right?

Reply:
Yes, there is no reason to use double quotes for that specific entry, I might have missed it. I generally only use quotes when special characters are used or to ensure the content is not confused with another type.